lib/scidata/imdb_reviews.ex

```diff
@@ -28,6 +36,8 @@ defmodule Scidata.IMDBReviews do
   `example_types` is the same argument in `download/2` but excludes `:unsup`
   because all unlabeled examples are in the training set.
   """
+  @spec download_test(sentiments, opts()) ::
+          %{review: any(), sentiment: any()}
```

Comment:
What if we document the default return types by documenting the behaviour with no opts? Something like:


```suggestion
  @spec download_test(sentiments) :: %{review: binary(), sentiment: -1 | 1}
  @spec download_test(sentiments, opts()) :: %{review: any(), sentiment: any()}
```

Reply:
We need to apply a similar change above to `download`. :)

Also, can we have unsup on both `download` and `download_test`?

Reply:
@josevalim good catch! `:unsup` only belongs with `download`.